### PR TITLE
Ice Bergs

### PR DESCRIPTION
**Pull request by WenXin20**
- Adds ice bergs to the ultraspace frozen oceans using SurfaceSystem mixins
  - Uses the tag `pokecube_world:is_iceberg_biome`
- Renamed tag to ~`pokecube_world:is_badlands_pillars_biome`~ from `is_custom_eroded_badlands`

### CODE REVIEW

**Comment by Thutmose** [comment]
the tag should still contain the word "eroded", so change it to `is_eroded_biome`

File: src/main/java/pokecube/world/WorldgenTags.java [hunk 1]
```diff
@@ -27,6 +27,9 @@ public static void initTags()
             Registry.CONFIGURED_STRUCTURE_FEATURE_REGISTRY,
             new ResourceLocation("pokecube_world", "mixin_restrictions/no_basalt"));
 
-    public static TagKey<Biome> IS_ERODED_BADLANDS = TagKey.create(Registry.BIOME_REGISTRY,
-            new ResourceLocation("pokecube_world", "is_custom_eroded_badlands"));
+    public static TagKey<Biome> IS_BADLANDS_PILLARS = TagKey.create(Registry.BIOME_REGISTRY,
```
Comment (Thutmose) [inline src/main/java/pokecube/world/WorldgenTags.java]: this should be `IS_ERODED_<name>`, changing `<name>` to `BIOME` might be fine, but it shouldn't be changed to pillars.

File: src/main/java/pokecube/mixin/features/SurfaceSystemMixin.java [hunk 2]
```diff
@@ -31,10 +31,15 @@ private void onGetBiome(BiomeManager manager, Registry<Biome> registry, boolean
             SurfaceRules.SurfaceRule surfacerules$surfacerule, BlockPos.MutableBlockPos blockpos$mutableblockpos1,
             int k, int l, int i1, int j1, int k1, Holder<Biome> holder)
     {
-        if (holder.is(WorldgenTags.IS_ERODED_BADLANDS))
+        if (holder.is(WorldgenTags.IS_BADLANDS_PILLARS))
```
Comment (Thutmose) [inline src/main/java/pokecube/mixin/features/SurfaceSystemMixin.java]: Also should include `ERODED`, not `PILLARS`

**Comment by WenXin20** [comment]
Changed to `is_eroded_biome`

**Pull request merged**